README.adoc

```diff
@@ -302,8 +302,8 @@ mvn compile
 ```
 Point to the
 {inv-url}[{inv-url}^] URL, check your Zipkin server, and sort the traces by newest first. You see a new trace record
-that is two spans long with one span for the [hotspot=40-42 file=1]`listContents()` JAX-RS method in the [hotspot file=1]`InventoryResource`
-class and another span for the [hotspot=43-45 file=0]`list()` method in the [hotspot file=0]`InventoryManager` class. Verify that these spans
+that is two spans long with one span for the [hotspot=listContents file=1]`listContents()` JAX-RS method in the [hotspot=InventoryResource file=1]`InventoryResource`
+class and another span for the [hotspot=list file=0]`list()` method in the [hotspot=InventoryManager file=0]`InventoryManager` class. Verify that these spans
```

Comment:
> [hotspot=InventoryManager file=0]`InventoryManager` 

should be: (since we're not highlighting the class, but the whole file)
> [hotspot file=0]`InventoryManager`